### DISCUSSION TITLE
[Combobox] Clear or Clear group missing

### DISCUSSION THREAD

**Discussion by rahulhsreedhar** [ing-bank/lion#1268] [General] [answered]
## Expected behavior

Combobox component should have clear or clear group.

## Actual Behavior

Clear or clear group is not found for combobox component.

**Comment by tlouisse** [comment, marked as answer]
Good point. We could add this to the ListboxMixin (that is the parent of the Combobox, SelectRich and Listbox).

It should have a `clear` method, since it is more like a Field than a FormGroup (think of native `<select>`, the options are not individual submittable elements). 
We might want to let the Listbox fully depend on LionField in the future and then we would get this api for free. 

For now, the quickest way of achieving this would be to add this to ListboxMixin:

```js
clear() {
  this.setCheckedIndex(-1);
}
```

For LionCombobox:

```js
clear() {
   super.clear();
   this._inputNode.value = '';
}
```

If you want, you could do a merge request for this.

**Reply by tlouisse** [reply]
Fixed by https://github.com/ing-bank/lion/pull/1315
